keywording/text.xml

```diff
@@ -338,12 +338,19 @@ So packages are not to be marked stable for one of these architectures.
 
 <p>
 If you maintain an architecture-independent package (data files, icons, pure
-Perl, ...) then you may request that your package be stabilized on all arches
+Python, ...) then you may request that your package be stabilized on all arches
```

Comment:
Is this some sort of python conspiracy? :P Maybe we should just say `interpreted-language code` or something.